### PR TITLE
Fix localstorage update on user update

### PR DESCRIPTION
**Pull request by ArsalanSavand**

### CODE REVIEW

File: app/basement/static/css/main.css [hunk 1]
```diff
@@ -217,10 +217,63 @@ md-toast {
...
 
 .nav-link-in-link {
   position: absolute;
   right: 0;
   margin-top: -26px;
-}
\ No newline at end of file
+}
+
+.myButt {
```
Comment (amirm3hdi) [inline app/basement/static/css/main.css]: @ArsalanSavand What's this? "butt" doesn't stand for button, Fix it

Comment (amirm3hdi) [inline app/basement/static/css/main.css]: Remove the complete custom CSS
@ArsalanSavand Try using bootstrap only

**Comment by amirm3hdi** [comment]
@ArsalanSavand This pull request is filled with off topic codes.

**Comment by amirm3hdi** [comment]
Make a new one

**Comment by Alir3z4** [comment]
@ArsalanSavand @amirm3hdi when you merge the pull request.
Please be sure to delete the branch as well.

**Pull request closed without merging**